resources/META-INF/plugin.xml

```diff
@@ -440,6 +432,8 @@
   </actions>
 
   <extensions defaultExtensionNs="com.intellij">
+     <!-- ApplicationInitializedListener  -->
```

Comment:
I think the comment is redundant since it's identical to the extension point name.